README.md

```diff
@@ -47,8 +47,8 @@ mdns while it is being delivered to mainline)
 6. Now you can run or build binary files of **ZSCU**:
     1. For running this application you need enter `npm run electron`
 command in "./src/"
-    2. For building binary files for the target OS you need run one of the
-folowing commands in "./src/":
+    2. You can building binary files only from target to target platform. For building binary files you need run one of the
```

Comment:
You can **build** binary files only from **the same platform type as the intended target**.